Sources/BaggageContext/BaggageContextCarrier.swift

```diff
@@ -209,12 +212,25 @@ extension DefaultContext {
     /// - Parameters:
     ///   - reason: Informational reason for developers, why a placeholder context was used instead of a proper one,
     /// - Returns: Empty "to-do" baggage context which should be eventually replaced with a carried through one, or `background`.
-    public static func TODO(logger: Logger, _ reason: StaticString? = "", function: String = #function, file: String = #file, line: UInt = #line) -> DefaultContext {
-        let baggage = Baggage.TODO(reason, function: function, file: file, line: line)
+    public static func TODO(logger: Logger, _ reason: StaticString? = "", function: StaticString = #function, file: StaticString = #file, line: UInt = #line) -> DefaultBaggageContext {
+        let baggage = DefaultBaggageContext.TODO(reason, function: function, file: file, line: line)
         #if BAGGAGE_CRASH_TODOS
         fatalError("BAGGAGE_CRASH_TODOS: at \(file):\(line) (function \(function)), reason: \(reason)", file: file, line: line)
         #else
-        return .init(baggage: baggage, logger: logger)
+        return .init(baggage: baggage as! Baggage, logger: logger)
```

Comment:
The need to force cast is very worrying, why?

Reply:
This was from the `-> Context` being returned, but switching it to `-> Self` makes this unnecessary